source/includes/authorization_configuration/_20-show.md.erb

```diff
@@ -0,0 +1,79 @@
+## Get an authorization configuration
+
+```shell
+$ curl 'https://ci.example.com/go/api/admin/security/auth_configs/ldap' \
+      -u 'username:password' \
+      -H 'Accept: <%= data.apis.versions.authorization_configurations %>'
+```
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: <%= data.apis.versions.authorization_configurations %>; charset=utf-8
+ETag: "cbc5f2d5b9c13a2cc1b1efb3d8a6155d"
+```
+
+```json
+{
+  "_links": {
+    "self": {
+      "href": "https://ci.example.com/go/api/admin/security/auth_configs/ldap"
+    },
+    "doc": {
+      "href": "https://api.gocd.io/#authorization-configuration"
+    },
+    "find": {
+      "href": "https://ci.example.com/go/api/admin/security/auth_configs/:auth_config_id"
+    }
+  },
+  "id": "ldap",
+  "plugin_id": "cd.go.authorization.ldap",
+  "properties": [
+    {
+      "key": "Url",
+      "value": "ldap://ldap.server.url:389"
+    },
+    {
+      "key": "SearchBase",
+      "value": "ou=users,ou=system"
+    },
+    {
+      "key": "ManagerDN",
+      "value": "uid=admin,ou=system"
+    },
+    {
+      "key": "SearchFilter",
+      "value": "uid"
+    },
+    {
+      "key": "Password",
+      "value": "secret"
```

Comment:
Not sure, but I recall that the configuration property representer provides `encrypted_value` based on the plugin's response as to whether the property is secure or not.